### PR TITLE
Update intro for Windows.md

### PR DESCRIPTION
**Pull request by celanthe**
Add tag: your-first-PR
#422

### CODE REVIEW

**Comment by KenanY** [comment]
@celanthe I think you should remove these plus-signs:

**Comment by celanthe** [comment]
@KenanY will do.

**Comment by ashleygwilliams** [comment]
hi @celanthe! a couple of things: 

- could you rebase this? let me know if you need help with that.
- it also appears you left a merge conflict in here? could you remove that?

File: content/getting-started/using-a-package.json.md [hunk 1]
```diff
@@ -115,7 +115,11 @@ list the packages you'd like to use in your `package.json` file. There are
 2 types of packages you can list:
 
 - `"dependencies"`: these packages are required by your application in production
+<<<<<<< HEAD
```
Comment (ashleygwilliams) [inline content/getting-started/using-a-package.json.md]: this should be deleted

**Comment by ashleygwilliams** [comment]
also! this is good advice for people on both windows and *nix systems. could you remove the part that specifically mentions windows?

**Comment by ashleygwilliams** [comment]
hey @celanthe did you see my commets? would love to merge if you could just update :)

**Comment by celanthe** [comment]
Hi @ashleygwilliams-- Can do! I'm running on a netbook right now, but I'll do my best to get to this in a few days. Thanks so much for getting back to me!

**Comment by ashleygwilliams** [comment]
no problem! @celanthe -- lemme know if you need help updating this!

**Comment by celanthe** [comment]
@ashleygwilliams -- Finally have a desktop rig again [had to ship it from the US to New Zealand.] Hoping to get to this and update w/your fixes today! Thank you so much.

**Comment by ashleygwilliams** [comment]
this is stale. closing.

**Pull request closed without merging**